src/components/Contentful/CenteredRichText.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+	<section-with-background-classic
+		:background-content="background"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid class="tw-grid-cols-12 tw-text-center">
```

Comment:
Does the grid give us anything here since we take up all of the columns?

Reply:
+1

Reply:
Cleaned these up.